extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -16,7 +16,16 @@
 
 import {isLayoutSizeDefined} from '../../../src/layout';
 import {user} from '../../../src/log';
+import {
+    installVideoManagerForDoc,
+} from '../../../src/service/video-manager-impl';
+import {VideoEvents} from '../../../src/video-interface';
+import {videoManagerForDoc} from '../../../src/services';
+import {assertAbsoluteHttpOrHttpsUrl} from '../../../src/url';
 
+/**
+ * @implements {../../../src/video-interface.VideoInterface}
```

Comment:
Since you now implement the video-interface, you will get `autoplay` for free. Please
- [ ] Update the validation file to allow for autoplay attribute (`attrs: { name: "autoplay" }`)
- [ ] Update the documentation MD file with:
```
**autoplay**

If this attribute is present, and the browser supports autoplay:

* the video is automatically muted before autoplay starts
* when the video is scrolled out of view, the video is paused
* when the video is scrolled into view, the video resumes playback
* when the user taps the video, the video is unmuted
* if the user has interacted with the video (e.g., mutes/unmutes, pauses/resumes, etc.), and the video is scrolled in or out of view, the state of the video remains as how the user left it.  For example, if the user pauses the video, then scrolls the video out of view and returns to the video, the video is still paused. 
```
- [ ] Update the example file to include autoplay and buttons for `play`, `pause`, `mute`, `unmute` actions. Please see `amp-youtube.amp.html` in `examples` for reference.